src/StorageSync/StorageSync/help/Unregister-AzStorageSyncServer.md

```diff
@@ -0,0 +1,236 @@
+---
+external help file: Microsoft.Azure.Commands.StorageSync.dll-Help.xml
+Module Name: Az.StorageSync
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.storagesync/unregister-Azstoragesyncserver
+schema: 2.0.0
+---
+
+# Unregister-AzStorageSyncServer
+
+## SYNOPSIS
+This command will use to unregister storage sync server.
+
+## SYNTAX
+
+### ObjectParameterSet (Default)
+```
+Unregister-AzStorageSyncServer [-Force] [-PassThru] [-AsJob] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Unregister-AzStorageSyncServer [-InputObject] <PSRegisteredServer> [-Force] [-PassThru] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Unregister-AzStorageSyncServer [-ResourceId] <String> [-Force] [-PassThru] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### StringParameterSet
+```
+Unregister-AzStorageSyncServer [-ResourceGroupName] <String> [-StorageSyncServiceName] <String>
```

Comment:
The parameter name for StorageSyncServiceName shoudl be 'Name' you can add an alias to StorageSyncServiceName, if  you like.

Reply:
StorageSyncServer is not same as StorageSyncService

---
In reply to: [255694600](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255694600) [](ancestors = 255694600)